Src/coffeescript/cql-execution/test/elm/datetime/test.coffee

```diff
@@ -702,6 +712,13 @@ describe 'DifferenceBetween Comparisons', ->
     should(@fortyDaysEqualToDaysBetween.exec(@ctx)).be.null
     @twentyFiveDaysGreaterThanDaysBetween.exec(@ctx).should.be.false()
 
+  it 'should properly determine that Sep to Dec is NOT <= 2 months', ->
+    @bonnieTestCase.exec(@ctx).should.be.false()
+
+  it 'should properly determine that Sep to Dec is NOT <= 2 months with 0 timezone offset (Zulu)', ->
```

Comment:
Where does the 'Zulu' come from?  The name of the patient where the bug was found?

Reply:
https://www.timeanddate.com/time/zones/z